### ISSUE TITLE
Can't Catch Pokemon

### ISSUE THREAD

**Issue by Anatine** [finndev/PokeBuddy#821]
I only want to catch Pokemon for the Stardust, but anytime I put it ONLY on that setting, it doesn't catch shit. Just constantly says "[Catch Pokemon] No Pokemon nearby"

**Comment by OneXV2** [comment]
soft ip banned

**Comment by Harlekin002** [comment]
Its possible that the bot work in a few teme ???

**Comment by OneXV2** [comment]
@Harlekin002 yeah by changing your ip. the bot will work again but that new ip will get banned. so far no solutions for this. your original ip will get unbanned but you gotta wait.

**Comment by chopperaus** [comment]
Because its only thing selected u dont move like if u have farm stops so nothing is spawning in the catch radius its "no pokemon nearby" hope that helps

**Comment by Anatine** [comment]
I'm not soft banned, I can catch pokemon if I select to also farm pokestops, but the thing is I ONLY want to farm pokemon. Just pokemon. It's no doing that. It's more concerned about hitting pokestops. Or it has issues detecting pokemon.

**Comment by chopperaus** [comment]
Like i said previously u dont move around when catch is only option selected.

**Comment by Anatine** [comment]
That's a problem that needs to be fixed then.
